S02-literals/string-interpolation.t

```diff
@@ -141,4 +141,13 @@ line 4
 
 }
 
+# https://github.com/rakudo/rakudo/issues/3070
+#?rakudo skip 'GH#3070 not yet fixed.'
+{
+    sub f(--> Nil) { say "{1}"; }
+    sub g(--> 2)   { say "{1}"; }
+    is f, '1', 'block interpolation works with signature-declared Nil return value';
```

Comment:
Not sure about this. `say` returns `True`, so f should actually return True, except it will return Nil because it's defined that way. Maybe you want to check for output of f, if that's what the bug is about.

Reply:
Oh right! Thanks, the routine shouldn't call say but rather return the string.

Reply:
Fixed and confirmed that "{1}" does indeed evaluate to the string "2".